extensions/wikia/DesignSystem/scripts/DesignSystemGlobalFooter.js

```diff
@@ -0,0 +1,12 @@
+require(['jquery'],
+	function ($) {
+		if ($.cookie('useskin') === 'oasis')  {
+			$('.mobile-site-link').click(mobileSiteLinkClicked);
```

Comment:
You can chain these, e.g.
```
$('.mobile-site-link')
    .click(mobileSiteLinkClicked)
    .css("display", "inline-block");
```
This way you do not evaluate the same selector twice (makes performance better, all the DOM operations are performance-costly).